packages/eslint-config-vtex/rules/typescript.js

```diff
@@ -26,6 +26,7 @@ module.exports = !hasTypescript
               '*/*/tsconfig{.eslint.json,.json}',
             ],
             projectFolderIgnoreList: [/node_modules/i],
+            allowAutomaticSingleRunInference: true,
```

Comment:
I think you should add one comment here with the link of the issue you found into typescript-eslint

Reply:
Added! 😄